pkg/parsing/impl/validator.go

```diff
@@ -48,7 +48,7 @@ func New(systemTablePrefixes []string, opts ...parsing.Option) (parsing.SQLValid
 func (pp *QueryValidator) ValidateCreateTable(query string, chainID tableland.ChainID) (parsing.CreateStmt, error) {
 	ast, err := sqlparser.Parse(query)
 	if err != nil {
-		return nil, &parsing.ErrInvalidSyntax{InternalError: err}
```

Comment:
I removed the `parsing.ErrInvalidSyntax`, now we just return a wrapper error since `err` can be an invalid syntax or another error such as multiple primary keys.